x-pack/platform/plugins/shared/alerting/common/routes/schedule/validation/validation_schedule/v1.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { validateSchedule } from './v1';
+
+describe('validateSchedule', () => {
+  beforeEach(() => jest.clearAllMocks());
+
+  it('validates duration and every correctly', () => {
+    expect(
+      validateSchedule({
+        duration: '1h',
+        recurring: { every: '2w' },
+      })
+    ).toBeUndefined();
+  });
+
+  it('throws error when duration in minutes and greater than interval', () => {
```

Comment:
We test it throws for every `d` and `w`. Maybe we can do the same for the other allowed `every` values, `M` and `y`.